dev/com.ibm.ws.transport.http.compression_fat/fat/src/com/ibm/ws/transport/http/HttpCompressionTests.java

```diff
@@ -1006,6 +1019,221 @@ public void testCompressionConfigPrefAlgoMisconfiguration() throws Exception {
 
     }
 
+    /**
```

Comment:
Do both of these tests need to run in LITE mode? Or can we place one of them in FULL mode?